<launch>
	<node pkg="swarm_control" type="car1_node" name="car1_node" output="screen">
		<param name="expectPos_carx_v1" value ="1.8"/>
		<param name="expectPos_cary_v1" value ="-1"/>
		<param name="expectPos_carx_v2" value ="3"/>
		<param name="expectPos_cary_v2" value ="0"/>
		<param name="vel_P" value ="3.0"/>
		<param name="vel_D" value ="0"/>
		<param name="vel_I" value ="0"/>
		<param name="w_P" value ="0.05"/>
	</node>

</launch>
